package.json

```diff
@@ -26,13 +26,7 @@
         "test:performance:best:ci": "nx test:best:ci @lwc/perf-benchmarks",
         "release:version": "./scripts/release/version.js"
     },
-    "//": {
-        "@babel/helper-explode-assignable-expression": "required for rollup-plugin-compat"
-    },
     "devDependencies": {
-        "@babel/core": "^7.22.5",
-        "@babel/helper-explode-assignable-expression": "^7.18.6",
-        "@babel/preset-env": "^7.22.5",
```

Comment:
Are these unused? Wasn't sure how it is related to this PR.

Reply:
Sorry, I made things confusing by merging this PR with #2948. I've removed those unrelated changes.